src/main/java/frc/robot/subsystems/gripper/GripperConstants.java

```diff
@@ -1,22 +1,38 @@
 package frc.robot.subsystems.gripper;
 
-import static frc.robot.subsystems.intake.IntakeConstants.*;
-
 import edu.wpi.first.units.*;
 import frc.robot.Constants;
 import lib.webconstants.LoggedTunableNumber;
 
 public class GripperConstants {
-    public static final Measure<Angle> INTAKE_ANGLE = null;
-    public static final Measure<Angle> OUTTAKE_ANGLE = null;
-    public static final double INTAKE_POWER = 0;
-    public static final double OUTTAKE_POWER = 0;
+    public static final Measure<Angle> INTAKE_ANGLE =
+            Units.Rotations.of(0); // TODO: replace with actual value
+    public static final Measure<Angle> OUTTAKE_ANGLE =
+            Units.Rotations.of(0); // TODO: replace with actual value
+    public static final double INTAKE_POWER = 0; // TODO; replace with actual value
+    public static final double OUTTAKE_POWER = 0; // TODO: replace with actual value
+    public static final double AMP_POWER_NORMAL = 0; // TODO: replace with actual value
+    public static final double AMP_POWER_REVERSE = 0; // TODO: replace with actual value
+    public static final double TRAP_POWER = 0; // TODO: replace with actual value
+    public static final double SHOOT_POWER = 0; // TODO: replace with actual value
     public static final Measure<Dimensionless> THRESHOLD = Units.Percent.of(0.02);
+    public static final Measure<Distance> GRIPPER_OUTTAKE_MIN_HEIGHT =
+            Units.Meters.of(0); // TODO: replace with actual value
+    public static final Measure<Distance> GRIPPER_LENGTH = Units.Meters.of(0.534_35);
```

Comment:
Do you think you can reach an accuracy of 10 micrometers? Impressive

Reply:
We can in the simulation, Also why not be as accurate as possible

Reply:
Just writing long numbers won't increase the accuracy of your subsystem. You are limited to the sensor's accuracy and the overall mechanical quality of the subsystem. It's not _bad_, it's just not beneficial.
I would use at most mm precision, but you can resolve this after reading, you don't have to change it. I just wanted to make sure you got my point.